From b35ef6c14ef5f8d90f6ee82e47b726044e5a2ecc Mon Sep 17 00:00:00 2001
From: Ankit Agrawal <ankit.agarwal@intel.com>
Date: Thu, 18 May 2023 17:51:50 +0530
Subject: [PATCH] Fixed for HIDL & HIDL Declaration in Manifest files.

Commenting hidl & aidl services declaration in manifest, as
it is already declared in their own service modules.

Tracked-On: OAM-106853
Signed-off-by: Tanuj Tekriwal <tanuj.tekriwal@intel.com>
---
 .../caas/framework_manifest.xml               | 28 ++++++++--------
 groups/device-specific/caas/manifest.xml      | 32 +++++++++----------
 2 files changed, 30 insertions(+), 30 deletions(-)

diff --git a/groups/device-specific/caas/framework_manifest.xml b/groups/device-specific/caas/framework_manifest.xml
index 45672a4..3ee7577 100644
--- a/groups/device-specific/caas/framework_manifest.xml
+++ b/groups/device-specific/caas/framework_manifest.xml
@@ -4,15 +4,15 @@
 {{#ota-update}}
 <manifest version="1.0" type="framework">
 {{/ota-update}}
-    <hal format="aidl">
+    <!--hal format="aidl">
         <name>android.hardware.identity</name>
         <version>4</version>
         <interface>
             <name>IIdentityCredentialStore</name>
             <instance>default</instance>
         </interface>
-    </hal>
-    <hal format="aidl">
+    </hal-->
+    <!--hal format="aidl">
         <name>android.hardware.security.keymint</name>
         <version>2</version>
         <interface>
@@ -27,26 +27,26 @@
             <name>IRemotelyProvisionedComponent</name>
             <instance>default</instance>
         </interface>
-    </hal>
-    <hal format="aidl">
+    </hal-->
+    <!--hal format="aidl">
         <name>android.hardware.security.secureclock</name>
         <interface>
             <name>ISecureClock</name>
             <instance>default</instance>
         </interface>
-    </hal>
-    <hal format="aidl">
+    </hal-->
+    <!--hal format="aidl">
         <name>android.hardware.security.sharedsecret</name>
         <interface>
             <name>ISharedSecret</name>
             <instance>default</instance>
         </interface>
-    </hal>
-    <hal format="aidl">
+    </hal-->
+    <!--hal format="aidl">
         <name>android.hardware.health</name>
         <version>1</version>
         <fqname>IHealth/default</fqname>
-    </hal>
+    </hal-->
     <hal format="hidl">
         <name>android.hardware.atrace</name>
         <transport>hwbinder</transport>
@@ -159,12 +159,12 @@
             <instance>default</instance>
         </interface>
     </hal>
-    <hal format="hidl">
+    <!--hal format="hidl">
         <name>android.hardware.drm</name>
         <transport>hwbinder</transport>
         <fqname>@1.4::ICryptoFactory/clearkey</fqname>
         <fqname>@1.4::IDrmFactory/clearkey</fqname>
-    </hal>
+    </hal-->
     <hal format="aidl">
         <name>android.hardware.usb</name>
         <version>1</version>
@@ -182,7 +182,7 @@
             <instance>external/0</instance>
         </interface>
     </hal>
-    <hal format="hidl">
+    <!--hal format="hidl">
         <name>android.hardware.boot</name>
         <transport>hwbinder</transport>
         <version>1.2</version>
@@ -190,7 +190,7 @@
             <name>IBootControl</name>
             <instance>default</instance>
         </interface>
-    </hal>
+    </hal-->
     <hal format="hidl">
         <name>android.hardware.renderscript</name>
         <transport arch="32+64">passthrough</transport>
diff --git a/groups/device-specific/caas/manifest.xml b/groups/device-specific/caas/manifest.xml
index 8737ef2..54bb15b 100644
--- a/groups/device-specific/caas/manifest.xml
+++ b/groups/device-specific/caas/manifest.xml
@@ -4,15 +4,15 @@
 {{#ota-update}}
 <manifest version="1.0" type="device">
 {{/ota-update}}
-    <hal format="aidl">
+    <!--hal format="aidl">
         <name>android.hardware.identity</name>
         <version>4</version>
         <interface>
             <name>IIdentityCredentialStore</name>
             <instance>default</instance>
         </interface>
-    </hal>
-    <hal format="aidl">
+    </hal-->
+    <!--hal format="aidl">
         <name>android.hardware.security.keymint</name>
         <version>2</version>
         <interface>
@@ -27,26 +27,26 @@
             <name>IRemotelyProvisionedComponent</name>
             <instance>default</instance>
         </interface>
-    </hal>
-    <hal format="aidl">
+    </hal-->
+    <!--hal format="aidl">
         <name>android.hardware.security.secureclock</name>
         <interface>
             <name>ISecureClock</name>
             <instance>default</instance>
         </interface>
-    </hal>
-    <hal format="aidl">
+    </hal-->
+    <!--hal format="aidl">
         <name>android.hardware.security.sharedsecret</name>
         <interface>
             <name>ISharedSecret</name>
             <instance>default</instance>
         </interface>
-    </hal>
-    <hal format="aidl">
+    </hal-->
+    <!--hal format="aidl">
         <name>android.hardware.health</name>
         <version>1</version>
         <fqname>IHealth/default</fqname>
-    </hal>
+    </hal-->
     <hal format="hidl">
         <name>android.hardware.thermal</name>
         <transport>hwbinder</transport>
@@ -112,14 +112,14 @@
            <instance>default</instance>
        </interface>
    </hal>
-    <hal format="aidl" optional="true">
+    <!--hal format="aidl" optional="true">
         <name>android.hardware.bluetooth.audio</name>
         <version>2</version>
         <interface>
             <name>IBluetoothAudioProviderFactory</name>
             <instance>default</instance>
         </interface>
-    </hal>
+    </hal-->
     <hal format="hidl">
         <name>android.hardware.media.omx</name>
         <transport>hwbinder</transport>
@@ -133,12 +133,12 @@
             <instance>default</instance>
         </interface>
     </hal>
-    <hal format="hidl">
+    <!--hal format="hidl">
         <name>android.hardware.drm</name>
         <transport>hwbinder</transport>
         <fqname>@1.4::ICryptoFactory/clearkey</fqname>
         <fqname>@1.4::IDrmFactory/clearkey</fqname>
-    </hal>
+    </hal-->
     <hal format="hidl">
         <name>android.hardware.camera.provider</name>
         <transport>hwbinder</transport>
@@ -149,7 +149,7 @@
             <instance>legacy/0</instance>
         </interface>
     </hal>
-    <hal format="hidl">
+    <!--hal format="hidl">
         <name>android.hardware.boot</name>
         <transport>hwbinder</transport>
         <version>1.2</version>
@@ -157,7 +157,7 @@
             <name>IBootControl</name>
             <instance>default</instance>
         </interface>
-    </hal>
+    </hal-->
     <hal format="hidl">
         <name>android.hardware.renderscript</name>
         <transport arch="32+64">passthrough</transport>
-- 
2.17.1

